eslint.config.ts

```diff
@@ -5,9 +5,13 @@ import vitest from "@vitest/eslint-plugin";
 import eslintPlugin from "eslint-plugin-eslint-plugin";
 import jsdoc from "eslint-plugin-jsdoc";
 import jsonc from "eslint-plugin-jsonc";
+import markdownLinks from "eslint-plugin-markdown-links";
 import n from "eslint-plugin-n";
+import nodeDependencies from "eslint-plugin-node-dependencies";
 import perfectionist from "eslint-plugin-perfectionist";
 import * as regexp from "eslint-plugin-regexp";
+import security from "eslint-plugin-security";
```

Comment:
I've never seen a codebase actually benefit from `eslint-plugin-security` (when `eslint-plugin-regexp` is in play).

* The security plugin is much more applicable for programs that run on user input, receive requests, etc. - not so much walled-garden lint plugins
* It's not very actively maintained

-1 from me on including it